server/controllers/menuControl.js

```diff
@@ -53,6 +53,31 @@ class MenuControl {
             }
         )
     }
+
+    static getOneMenu(req,resp) {
+        db.query(
+            MenuQuery.getOneMenuIDQuery(req.params.id),
+            (err, res) => {
+                const [menu] = res.rows;
+                if (menu=="" || menu==null) {
+                    return resp.status(404).send({
+                        status: 'error',
+                        message: 'Menu Not Found'
+                    })
+                }
+                return resp.status(200).send({
+                    status: 'success',
+                    message: 'Returned one menu',
+                    menu: {
```

Comment:
why are you breaking down the object like this? Why not just specify the columns returned from the query, so you don't have to break down the object here?

Reply:
Okay, I'd do that now